docs/reference/cat/indices.asciidoc

```diff
@@ -75,6 +75,14 @@ include::{es-repo-dir}/rest-api/common-parms.asciidoc[tag=help]
 include::{es-repo-dir}/rest-api/common-parms.asciidoc[tag=include-unloaded-segments]
 
 include::{es-repo-dir}/rest-api/common-parms.asciidoc[tag=local]
++
+--
+`local`::
+(Optional, boolean) If true, the request retrieves information from the local node
+only. Defaults to false, which means information is retrieved from the master node.
+deprecated::[8.0,This parameter does not cause this API to act locally. It will be
+removed in version 8.0.]
+--
```

Comment:
This creates two entries for the `local` parameter, one pulled from the `common-parms` file and this one. I'll solicit some input from our documentation writers on how best to handle this.

Reply:
@jrodewig, do you have any suggestions on how to handle this documentation case where a parameter pulled in a from a common file needs to be marked as deprecated on a specific API?

Reply:
I left a suggestion here: https://github.com/elastic/elasticsearch/pull/62198#discussion_r492857985